### PR TITLE
Feat/refactoring

### PR DESCRIPTION
**Pull request by L-M-K-B**

### CODE REVIEW

File: src/misc/styleComponents.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  @media (min-width: 375px) and (max-width: 499px) {
+    width: ${AppWidthM};
+  }
+  @media (min-width: 500px) {
+    width: ${AppWidthL};
+  }
+`;
+
+export const ContentContainer = styled.div`
+  display: flex;
```
Comment (markmattjohnson) [inline src/misc/styleComponents.js]: maybe section instead of div

File: src/misc/styleComponents.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    flex-direction: column;
+    align-items: center;
+  }
+`;
+
+export const BoxContainer = styled.div`
+  width: 100%;
+  margin-bottom: 30px;
+  padding: 1px 0;
+  border-radius: ${borderRadius};
```
Comment (markmattjohnson) [inline src/misc/styleComponents.js]: Please check padding :)

File: src/recipes-detailed/recipeDetailedStyle.js [hunk 3]
```diff
@@ -38,70 +12,102 @@ export const Favorite = styled.div`
...
+  }
 `;
 export const StyledTagList = styled.div`
-  grid-column: 1 / span 12;
-  grid-row: 7 / span 1;
   align-self: center;
+
+  @media (min-width: 375px) {
+    grid-column: 1 / span 12;
+    grid-row: 7 / span 1;
```
Comment (markmattjohnson) [inline src/recipes-detailed/recipeDetailedStyle.js]: why grid-row: 7 which grid did you use?

File: src/recipes-overview/recipesOverviewStyle.js [hunk 4]
```diff
@@ -123,14 +137,24 @@ export const Name = styled.p`
...
   margin: 0 0 2px;
 `;
 export const Time = styled.div`
-  margin: 0 0 15px 0;
+  @media (max-width: 374px) {
+    margin: 0 10px 15px 0;
+  }
+  @media (min-width: 375px) {
+    margin: 0 0 15px 0;
+  }
```
Comment (markmattjohnson) [inline src/recipes-overview/recipesOverviewStyle.js]: margin-right

File: src/recipes-overview/RecipesOverviewPage.js [hunk 5]
```diff
@@ -65,13 +66,17 @@ export default function RecipesOverviewPage({
...
         srcIconChef={IconChef}
       />
-    </GridOverview>
+    </GridHeaderSmall>
   );
 }
+
+/*
+
+*/
```
Comment (markmattjohnson) [inline src/recipes-overview/RecipesOverviewPage.js]: Don't use empty comments

File: src/misc/styleComponents.js (hunk 2 above)
Comment (L-M-K-B) [reply src/misc/styleComponents.js]: solved

File: src/recipes-detailed/recipeDetailedStyle.js (hunk 3 above)
Comment (L-M-K-B) [reply src/recipes-detailed/recipeDetailedStyle.js]: solved

File: src/recipes-overview/recipesOverviewStyle.js (hunk 4 above)
Comment (L-M-K-B) [reply src/recipes-overview/recipesOverviewStyle.js]: solved

File: src/recipes-overview/RecipesOverviewPage.js (hunk 5 above)
Comment (L-M-K-B) [reply src/recipes-overview/RecipesOverviewPage.js]: solved

**Pull request merged**